partner_credit_limit/model/__init__.py

```diff
@@ -0,0 +1,11 @@
+# -*- encoding: utf-8 -*-
+############################################################################
+#    Module Writen For Odoo, Open Source Management Solution
+#
+#    Copyright (c) 2011 Vauxoo - http://www.vauxoo.com
+#    All Rights Reserved.
+#    info Vauxoo (info@vauxoo.com)
+#    coded by: hugo@vauxoo.com
+#    planned by: Nhomar Hernandez <nhomar@vauxoo.com>
+############################################################################
+from . import accout_invoice, partner  # sale
```

Comment:
@moylop260 , yes, you comment me that this validación feature, only will be applied in account invoice at the moment, I'll remove the #sale commet